app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -95,4 +99,19 @@ class SyncIndicatorViewController @Inject constructor(
       else -> Synced(timeSinceLastSync)
     }
   }
+
+  private fun startSync(events: Observable<UiEvent>): Observable<UiChange> {
+    fun errorsStream() = dataSync
```

Comment:
This is a `fun` and not a `val` because while writing tests for this controller, I realized that if it is a `val` then `dataSync.streamSyncErrors()` will have to be configured in `setUp()` for every test by returning `Observable.never()`
And in doing so, it will not be overridden when we actually want to test this function.

Reply:
Sidenote:

Is my understanding correct that you wanted the `val`'s value to be lazily executed? In my experience, lambdas work nicely for this case.

Reply:
Yes. You are right. That is what I want.

Reply:
I am assuming you will make this change?

Reply:
Yes I will.